frontend/app/routes/public/apply/$id/child/phone-number.tsx

```diff
@@ -0,0 +1,175 @@
+  const { t } = useTranslation(handle.i18nNamespaces);
+  const { defaultState, editMode } = loaderData;
+  const fetcher = useFetcher<typeof action>();
+  const isSubmitting = fetcher.state !== 'idle';
+
+  const errors = fetcher.data?.errors;
+  const errorSummary = useErrorSummary(errors, {
+    phoneNumber: 'phone-number',
+    phoneNumberAlt: 'phone-number-alt',
+  });
+
+  return (
+    <div className="max-w-prose">
+      <p className="mb-4 italic">{t('apply:optional-label')}</p>
+      <errorSummary.ErrorSummary />
+      <fetcher.Form method="post" noValidate>
+        <CsrfTokenInput />
+        <div className="mb-6">
+          <p className="mb-4">{t('apply-child:phone-number.help-message')}</p>
+          <div className="grid items-end gap-6">
+            <InputPhoneField
+              id="phone-number"
+              name="phoneNumber"
+              type="tel"
+              inputMode="tel"
+              className="w-full"
+              autoComplete="tel"
+              defaultValue={defaultState.phoneNumber ?? ''}
+              errorMessage={errors?.phoneNumber}
+              label={t('apply-child:phone-number.phone-number')}
+              maxLength={100}
+              aria-describedby="adding-phone"
+            />
+            <InputPhoneField
+              id="phone-number-alt"
+              name="phoneNumberAlt"
+              type="tel"
+              inputMode="tel"
+              className="w-full"
+              autoComplete="tel"
+              defaultValue={defaultState.phoneNumberAlt ?? ''}
+              errorMessage={errors?.phoneNumberAlt}
+              label={t('apply-child:phone-number.phone-number-alt')}
+              maxLength={100}
+              aria-describedby="adding-phone"
+            />
+          </div>
+        </div>
+        {editMode ? (
+          <div className="flex flex-wrap items-center gap-3">
+            <Button id="save-button" name="_action" value={FORM_ACTION.save} variant="primary" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Child:Save - Phone Number click">
+              {t('apply-child:phone-number.save-btn')}
+            </Button>
+            <ButtonLink id="cancel-button" routeId="public/apply/$id/child/review-adult-information" params={params} disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Child:Cancel - Phone Number click">
+              {t('apply-child:phone-number.cancel-btn')}
+            </ButtonLink>
+          </div>
+        ) : (
+          <div className="mt-8 flex flex-row-reverse flex-wrap items-center justify-end gap-3">
+            <LoadingButton variant="primary" id="continue-button" loading={isSubmitting} endIcon={faChevronRight} data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form-Child:Continue - Phone number click">
+              {t('apply-child:new-or-existing-member.continue-btn')}
```

Comment:
Just noticed this (will probably have to be addressed in all apply flows)

```suggestion
              {t('apply-child:phone-number.continue-btn')}
```